internal/resource/resource.go

```diff
@@ -66,6 +81,10 @@ func (r Resource) Create(ctx context.Context, request resource.CreateRequest, re
 		return
 	}
 
+	if contains(r.FailOnCreate, resource.GetId()) {
```

Comment:
```suggestion
	if slices.Contains(r.FailOnCreate, resource.GetId()) {
```

As of Go 1.21, you can use the slices package instead of implementing the function yourself.

Reply:
Done!